internal/server/users.go

```diff
@@ -107,3 +107,27 @@ func (a *API) UpdateUser(c *gin.Context, r *api.UpdateUserRequest) (*api.User, e
 func (a *API) DeleteUser(c *gin.Context, r *api.Resource) (*api.EmptyResponse, error) {
 	return nil, access.DeleteIdentity(c, r.ID)
 }
+
+func (a *API) DeleteUser2(c *gin.Context, r *api.Resource) (*api.EmptyResponse, error) {
```

Comment:
Is this just a temporary function name showing how you would like to update `DeleteUser`?

Reply:
yes! so you can see both :)